tck/features/uncategorized/FunctionsAcceptance.feature

```diff
@@ -30,49 +30,21 @@
 
 Feature: FunctionsAcceptance
 
-  Scenario: Functions should return null if they get path containing unbound
-    Given any graph
-    When executing query:
-      """
-      WITH null AS a
-      OPTIONAL MATCH p = (a)-[r]->()
-      RETURN size(nodes(p)), type(r), nodes(p), relationships(p)
-      """
-    Then the result should be, in any order:
-      | size(nodes(p)) | type(r) | nodes(p) | relationships(p) |
-      | null           | null    | null     | null             |
-    And no side effects
-
-  Scenario: `properties()` on a node
+  Scenario: Run coalesce
```

Comment:
The scenario "Run coalesce" should be in "Conditional1 - Coalesce Expression" by commit https://github.com/RedisGraph/openCypher/commit/35f7e54042eadeac33c5986c91cf1e0b1a528a70

It seems from https://github.com/RedisGraph/openCypher/commits/elemntary_graph_operations that you have rebase the our branch, however it still wants to duplicate this scenario back into uncategorized/FunctionsAcceptance. Please double check.

Reply:
removed